source/features/latest-tag-button.tsx

```diff
@@ -18,45 +30,83 @@ const getLatestTag = cache.function(async (): Promise<string | false> => {
 			}) {
 				nodes {
 					name
+					target {
+						oid
+						... on Commit {
+							committedDate
+						}
+					}
 				}
 			}
 		}
 	`);
 
-	const tags: string[] = repository.refs.nodes.map((tag: {name: string}) => tag.name);
+	const tags: TagInfo[] = repository.refs.nodes.map((tag: {name: string; target: {oid: string; committedDate: string}}) => new TagInfo(tag));
 	if (tags.length === 0) {
 		return false;
 	}
 
 	// If all tags are plain versions, parse them
-	if (tags.every(tag => /^[vr]?\d/.test(tag))) {
-		return tags.sort(compareVersions).pop()!;
+	if (tags.every(tag => /^[vr]?\d/.test(tag.name))) {
+		return tags.sort((o1, o2) => compareVersions(o1.name, o2.name)).pop()!;
 	}
 
 	// Otherwise just use the latest
 	return tags[0];
 }, {
 	expiration: 1,
-	cacheKey: () => __featureName__ + ':' + getRepoURL()
+	cacheKey: () => __featureName__ + '_tags:' + getRepoURL()
+});
+
+const masterHasCommitsAfter = cache.function(async (commitDate: string): Promise<boolean> => {
+	// Find out whether we have commits after a certain date
+	const repository = await api.v4(`
+		repository(${getRepoGQL()}) {
+			object(expression: "master") {
+				... on Commit {
+				    oid
+				    history(first: 10, since: "${commitDate}") {
+					    nodes {
+					        committedDate
+					    }
+				    }
+				}
+			}
+		}
+	`);
+	const commitList = repository.repository.object.history.nodes;
+	// The commit on this exact date is always returned - we only want to know whether there was another commit afterwards
+	return commitList && commitList.length > 1;
+}, {
+	expiration: 1,
+	cacheKey: () => __featureName__ + '_commits:' + getRepoURL()
 });
 
-function getTagLink(latestRelease: string): HTMLAnchorElement {
+async function getTagLink(latestRelease: TagInfo): Promise<HTMLAnchorElement> {
 	const link = <a className="btn btn-sm btn-outline tooltipped tooltipped-ne ml-2">{tagIcon()}</a> as unknown as HTMLAnchorElement;
 
 	const currentBranch = getCurrentBranch();
-
-	if (currentBranch === latestRelease) {
-		link.classList.add('disabled');
-		link.setAttribute('aria-label', 'You’re on the latest release');
+	if (currentBranch === latestRelease.name) {
+		const [isBleedingEdge] = await Promise.all([
+			masterHasCommitsAfter(latestRelease.date)
+		]);
+		if (isBleedingEdge) {
+			link.setAttribute('aria-label', 'Current branch is bleeding edge');
```

Comment:
```suggestion
			link.setAttribute('aria-label', 'The current branch is ahead of the latest release');
```

cc @sindresorhus suggestions?

Reply:
> ${branch} is ${ahead_by} commits ahead of the latest release

👍